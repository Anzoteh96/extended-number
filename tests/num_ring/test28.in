619812354822987122230338154423642299608523866172742009527323690641738515824632798361
597559716050649698422956767273630043245188745712793371710876046145826125420130006013
-98363985416448093670914222354904711506286563277337310272224691887545553114230731594
