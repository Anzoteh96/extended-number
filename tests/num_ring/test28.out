Enter the number base b:
Enter two numbers n and m:
b is 619812354822987122230338154423642299608523866172742009527323690641738515824632798361
n mod b is 597559716050649698422956767273630043245188745712793371710876046145826125420130006013
m mod b is 521448369406539028559423932068737588102237302895404699255098998754192962710402066767
n + m is 499195730634201604752042544918725331738902182435456061438651354258280572305899274419
n - m is 76111346644110669863532835204892455142951442817388672455777047391633162709727939246
n * m is 446482683787522491326877660770188025199075579091761529214823961389175672800300619693
n ** m is 433050286490572538074943359505942835031361070920345397760190491142953935553321973021
